core/modules/css/KvInterface.h

```diff
@@ -42,18 +42,28 @@ namespace css {
 
 class KvInterface {
 public:
+
+    /**
+     * Sets a max length for kvKey.
+     * If it is changed it must also be changed in the
+     * CssData SQL Schema as well.
+     */
+    const unsigned int kMaxKeyLength = 255;
```

Comment:
This is declared as data member of a class, which we do not want or need. Either declare is as static or make it an enum.

Our coding guidelines say:
> Named constants (including enumeration values) MUST be all uppercase using underscore to separate words

so the name should look like `MAX_KEY_LENGTH`